providers/build.info

```diff
@@ -162,5 +162,3 @@ ENDIF
 # include all the object files that are needed.
 $NULLGOAL=../libcrypto
 SOURCE[$NULLGOAL]=nullprov.c prov_running.c
-
-SOURCE[$LIBDEFAULT]=prov_running.c
```

Comment:
Shouldn't prov_running.c really end up in `SOURCE[$LIBCOMMON]`?

Reply:
It doesn't belong in the FIPS provider.

Reply:
There would be a conflict with libfips. Some of the functions are reimplemented in fips/self_test.c.

Reply:
Ah, right, especially since $LIBCOMMON is used as SOURCE for the FIPS module, rather than a mere DEPEND...
Never mind then